bitrise.yml

```diff
@@ -129,16 +127,25 @@ workflows:
       - deploy-to-bitrise-io@2: { }
   run-instrumentation-tests:
     before_run:
+      - start_emulator
       - prepare_all
     after_run:
       - conclude_all
     steps:
-      - avd-manager@1: { }
-      - wait-for-android-emulator@1: { }
+      - wait-for-android-emulator@1:
+          inputs:
+            - boot_timeout: 600
       - script-runner@0:
           title: Execute instrumentation tests
           inputs:
             - file_path: ./scripts/execute_instrumentation_tests.sh
+  start_emulator:
+    steps:
+      - avd-manager@1:
+          inputs:
+            - profile: "pixel_3a"
+            - api_level: 30
+            - tag: "google_apis_playstore"
```

Comment:
Extracted emulator creation to a workflow so that 1. we can run it on the `before_run` section and 2. Emulator creation is consistent across instrumentation tests. But let me know if it does not make sense for Payments!